docs/redis-session-cache.md

```
@@ -0,0 +1,124 @@
+# Redis Session Cache
+
+This application uses Redis (via Vercel KV) to cache session validation results, significantly improving performance by reducing FileMaker database queries.
+
+## How It Works
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)